src/libraries/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -320,6 +319,31 @@ public void StartStop()
             Assert.Null(Activity.Current);
         }
 
+        /// <summary>
+        /// Tests Activity.IsStopped
+        /// </summary>
+        [Fact]
+        public void IsStoppedTest()
+        {
+            var activity = new Activity("activity");
```

Comment:
```suggestion
            using var activity = new Activity("activity");
```
?

Reply:
Dispose wouldn't add anything here as I am manually stopping the Activity. What is the value of having `using`?

Reply:
If an assert fails before you call stop causing an exception to be thrown and the activity not disposed.  You know the impact best, though, so up to you.